invenio_vocabularies/contrib/names/schema.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Names schema."""
+
+from functools import partial
+
+from flask_babelex import lazy_gettext as _
+from invenio_records_resources.services.records.schema import BaseRecordSchema
+from marshmallow import Schema, ValidationError, fields, post_load, \
+    validates_schema
+from marshmallow_utils.fields import IdentifierSet, SanitizedUnicode
+from marshmallow_utils.schemas import IdentifierSchema
+
+from .config import names_schemes
+
+
+# FIXME: this is a dup from rdm-records, shall we add it in the aff vocab?
```

Comment:
Indeed it's weird that this is not in this module (since it's pretty much a Service schema validation rule for creating affiliations). Maybe, shelve and we see it can be addressed separately.

Reply:
done https://github.com/inveniosoftware/invenio-vocabularies/issues/98